assembl/static2/js/app/components/administration/survey/step3.jsx

```diff
@@ -38,17 +40,24 @@ class Step3 extends React.Component<Props, State> {
     this.setState({ translate: shouldTranslate });
   };
 
+  handleAnonymousChange = () => {
+    this.setState({ isAnonymous: !this.state.isAnonymous });
```

Comment:
You souldn't access this.state like that. You should use this syntax: `this.setState((prevState) => ({isAnonymous: !prevState.isAnonymous}))`

Reply:
More info here: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous

Reply:
Thanks for the heads up. Changes made.